docs/TEAMS.md

```diff
@@ -0,0 +1,202 @@
+The Organic Maps is led by contributors from across the community. Different functional domains may periodically establish, develop, or dissolve their own leadership structures, whether formal or informal.
+
+The file contains the functional project structure. The structure of GitHub teams must match the content of this file. Any changes to memberships must be performed via a PR to this file.
+
+- [Android Team](#android-team)
+- [iOS Team](#ios-team)
+- [Map Data Team](#map-data-team)
+- [Map Styles Team](#map-styles-team)
+- [DevOps Team](#devops-team)
+- [Design Team](#design-team)
+- [Website Team](#website-team)
+- [Growth Team](#growth-team)
+- [Translation Team](#translation-team)
+- [User Support Team](#user-support-team)
+- [Legal Team](#legal-team)
+
+## Android Team
+
+[@organicmaps/android](https://github.com/orgs/organicmaps/teams/android/members)
+
+**Scope**:
+
+Android application development.
+
+**Members:**
```

Comment:
Do we need to specify the members of each team here? It may be hard to maintain the list in the future. All members are visible on GitHub.

Reply:
The Github Android team is for (technical) reviewers.
The list in TEAMS could be more generic including all active contributors and possibly what area they specialize in (AA, UI, etc) and what permissions do they hold.

Reply:
There is no issue with adding more specific android-auto team or subteam.

Reply:
Added Android Auto.